lib/plugins/aws/provider.js

```diff
@@ -551,6 +551,7 @@ class AwsProvider {
           },
           awsLambdaRuntime: {
             enum: [
+              'dotnet6',
```

Comment:
Is there a documentation reference that this is proper syntax for this runtime? I looked into CloudFormation docs but it seems they're not updated yet

Reply:
Code snippet from the _ASP.NET Core minimal APIs_ section of the attached blog post:

```json
   "AspNetCoreFunction": {
      "Type": "AWS::Serverless::Function",
      "Properties": {
        "Handler": "AspNetCoreMinimalApiExample",
        "Runtime": "dotnet6"
        "MemorySize": 256,
        "Timeout": 30,
        "Role": null,
        "Policies": [
          "AWSLambda_FullAccess",
          "AmazonS3ReadOnlyAccess"
        ],
   }
```

It's also documented [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-csharp.html).

Reply:
Perfect, thanks a lot 🙇